src/pages/guides/api/journaling_api.md

```diff
@@ -73,15 +73,15 @@ The journaling endpoint URL, when called without any query parameters fetches th
 
 For example:
 
-```
+```bash
 curl -X GET \
   https://events.adobe.io/events/organizations/23294/integrations/54108/f89067f2-0d50-4bb2-bf78-209d0eacb6eb \
   -H "x-ims-org-id: 4CC7D9704674CFB2138A2C54@AdobeOrg" \
   -H "Authorization: Bearer $USER_TOKEN" \
   -H "x-api-key: $API_KEY"
 ```
 
-```
+```json
```

Comment:
why `json` here?

Reply:
Did a regex find and replace. Some HTTP requests are just 204 so no content to display, but since this is only for syntax highlighting it didn't make a difference. Apparently there is no `http` language for syntax highlighting.

Reply:
ok cool

Reply:
I have marked http code blocks that have no json response with `http` even though it doesn't make a difference to the syntax highlighting.